### ISSUE TITLE
paddleocr预训练模型加载问题

### ISSUE THREAD

**Issue by H-eaven-1** [PaddlePaddle/PaddleOCR#9767]
训练检测模型的过程中，加载MobileNetV3_large_x1_0_pretrained.pdparams模型时出错,显示类似下面的问题
[2023/04/19 14:37:51] ppocr WARNING: The pretrained params conv1.conv.weight not in model
[2023/04/19 14:37:51] ppocr WARNING: The pretrained params conv1.bn.weight not in model
[2023/04/19 14:37:51] ppocr WARNING: The pretrained params conv1.bn.bias not in model
[2023/04/19 14:37:51] ppocr WARNING: The pretrained params conv1.bn._mean not in model
[2023/04/19 14:37:51] ppocr WARNING: The pretrained params conv1.bn._variance not in model

配置文件如下：
Global:
  use_gpu: true
  use_xpu: false
  use_mlu: false
  epoch_num: 300
  log_smooth_window: 20
  print_batch_step: 30
  save_model_dir: E:/output/db_mv3
  save_epoch_step: 40
  eval_batch_step: [0, 500]
  cal_metric_during_train: False                   
  pretrained_model: E:/pretrain_models/MobileNetV3_large_x1_0_pretrained
  checkpoints:
  save_inference_dir:
  use_visualdl: True
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt

Architecture:
  model_type: det
  algorithm: DB
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 1.0
    model_name: large
  Neck:
    name: DBFPN
    out_channels: 256
  Head:
    name: DBHead
    k: 50

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: E:/paddleocr_out/train_data/det/
    label_file_list:
      - E:/paddleocr_out/train_data/det/label.txt
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - IaaAugment:
          augmenter_args:
            - { 'type': Fliplr, 'args': { 'p': 0.5 } }
            - { 'type': Affine, 'args': { 'rotate': [-10, 10] } }
            - { 'type': Resize, 'args': { 'size': [0.5, 3] } }
      - EastRandomCropData:
          size: [640, 640]
          max_tries: 50
          keep_ratio: true
      - MakeBorderMap:
          shrink_ratio: 0.4
          thresh_min: 0.3
          thresh_max: 0.7
      - MakeShrinkMap:
          shrink_ratio: 0.4
          min_text_size: 8
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask'] # the order of the dataloader list
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 8
    num_workers: 1
    use_shared_memory: True

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: E:/paddleocr_out/train_data/det/
    label_file_list:
      - E:/paddleocr_out/train_data/det/label1.txt
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - DetLabelEncode: # Class handling label
      - DetResizeForTest:
          image_shape: [736, 1280]
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'shape', 'polys', 'ignore_tags']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 1
    use_shared_memory: True

在backbone部分的选择和预训练模型是对应的，但是为什么会有参数匹配不上的问题呢

**Comment by raderqi** [comment]
我看他们官方出版的教程里也有这个warning，但是他们是能够正常训练下去的

**Comment by zhwshr** [comment]
> 我看他们官方出版的教程里也有这个warning，但是他们是能够正常训练下去的

能训练，但是预训练模型参数加载不进去，模型训练的时间会长一些，效果也不太好
